### PR TITLE
Add Voice API

### PR DESCRIPTION
**Pull request by dstotijn**
This PR adds Voice API endpoints to the PHP client.

### CODE REVIEW

**Comment by EddieSeli** [comment]
Is there a way you can add contact through the PHP API?. Because when I was testing that I got the error "You are not authorized to perform this operation"

**Comment by dstotijn** [comment]
@EddieSeli: Is that related to this PR? Otherwise better suited for #45, which deals with contacts and groups.

**Comment by EddieSeli** [comment]
Okay, thanks. Will check that out

File: examples/voice-recordings-download.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+require_once(__DIR__ . '/../autoload.php');
+
+$messageBird = new \MessageBird\Client('YOUR_ACCESS_KEY'); // Set your own API access key here.
+
+try {
+    $data = $messageBird->voiceRecordings->download('c226420d-f107-4db1-b2f9-4646656a90bc', '4f5ab5f4-c4b6-4586-9255-980bb3fd7336', 'a94f7d51-19b5-4eb8-9e8e-90fce490a577'); // Set call, leg and recording id here
+    echo sprintf("Received %d bytes.", mb_strlen($data));
```
[the remark this answers is not in the document]
Comment (dstotijn) [reply examples/voice-recordings-download.php]: Done

File: examples/voice-transcriptions-download.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+require_once(__DIR__ . '/../autoload.php');
+
+$messageBird = new \MessageBird\Client('YOUR_ACCESS_KEY'); // Set your own API access key here.
+
+try {
+    $data = $messageBird->voiceTranscriptions->download('c226420d-f107-4db1-b2f9-4646656a90bc', '4f5ab5f4-c4b6-4586-9255-980bb3fd7336', 'a94f7d51-19b5-4eb8-9e8e-90fce490a577', '44e73d1f-201d-4a7d-963a-9d76bbca6c4f'); // Set call, leg, recording and transcription id here
+    echo sprintf("Received %d bytes.", mb_strlen($data));
```
[the remark this answers is not in the document]
Comment (dstotijn) [reply examples/voice-transcriptions-download.php]: Done

**Comment by Dirk94** [comment]
`Client->voiceCallFlows()->getList()` returns a `BaseList` object. `BaseList` expects the API to return 
```
{ 
  "offset": 0,
  "limit": 20,
  "count": 1,
  "totalCount": 1,
  "links": {
    "first": "url-here",
    "previous": null,
    "next": null,
    "last": "url-here"
  }
  ...
}
```

However the voice API returns 
```
{
  ...
  "pagination": {
    ...
  }
}
```

Therefore the `BaseList` attributes (offset, limit, links, etc) are null and the pagination attributes are ignored.

**Comment by dstotijn** [comment]
@Dirk94 Pushed a fix for this. Because Voice API currently has different pagination params (e.g. no `offset`) and also no `links` param, I've introduced a new `BaseList` for Voice specific resources.

**Comment by samwierema** [comment]
@Dirk94 tests are still failing. It seems like the PR uses the new array notation `[]`. Can you fix?

**Comment by samwierema** [comment]
@dstotijn can you do review of the changes since this week?

**Comment by dstotijn** [comment]
OK, looks good!

**Pull request merged**